internal/services/compute/shared_image_resource.go

```diff
@@ -438,3 +429,48 @@ func flattenGalleryImagePurchasePlan(input *compute.ImagePurchasePlan) []interfa
 		},
 	}
 }
+
+func expandGalleryImageFeatures(d *pluginsdk.ResourceData) *[]compute.GalleryImageFeature {
+	var features []compute.GalleryImageFeature
+
+	if d.Get("trusted_launch_enabled").(bool) {
+		features = append(features, compute.GalleryImageFeature{
+			Name:  utils.String("SecurityType"),
+			Value: utils.String("TrustedLaunch"),
+		})
+	}
+
+	if d.Get("support_accelerated_network").(bool) {
```

Comment:
(as above) can we move this into the Create function?

Reply:
Done